toolkit.rb

```diff
@@ -11,7 +11,7 @@
 args = {}
 args_array.each do |arg|
   arg_name  = arg.split("=").first.to_sym
-  arg_value = arg.split("=").last
+  arg_value = arg.gsub(arg.split("=").first, "").delete_prefix("=")
```

Comment:
was there a special case that this fixed? We haven't had any report problems with the current arg logic...